File: frogment/src/main/java/ch/grze/frogment/core/backstack/BackStackChangeListener.java
package ch.grze.frogment.core.backstack;

import ch.grze.frogment.frogment.FrogmentInterface;

public interface BackStackChangeListener {
    void onFrogmentPushed(FrogmentInterface frogment);
    void onFrogmentPopped(FrogmentInterface frogment);
    void onBackStackEmpty();
}

File: frogment/src/main/java/ch/grze/frogment/core/backstack/BackStackChangeListener.kt
package ch.grze.frogment.core.backstack

import ch.grze.frogment.frogment.FrogmentInterface

interface BackStackChangeListener {
    fun onFrogmentPushed(frogment: FrogmentInterface)
    fun onFrogmentPopped(frogment: FrogmentInterface)
    fun onBackStackEmpty()
}
